.github/workflows/maven.yml

```diff
@@ -4,8 +4,6 @@
 name: Java CI with Maven
 
 on:
-  push:
```

Comment:
why?

Reply:
lol, i thought i pushed it directly to master, is not part of this PR, will remove